spec/deqm_test_kit/v5.0.0/evaluate_spec.rb

```diff
@@ -602,7 +602,7 @@ def run(runnable, inputs = {})
       )
         .to_return(status: 400, body: error_outcome.to_json)
 
-      result = run(test, url:, measure_id:, patient_id:, period_start:)
+      result = run(test, url:, measure_id:, patient_id:, period_start:, period_end: '2019-12-31')
```

Comment:
```suggestion
      result = run(test, url:, measure_id:, patient_id:, period_start:)
```
This can be removed now that the inputs were moved back